app/connectors/aws/S3FileManager.scala

```diff
@@ -103,7 +103,7 @@ class S3FileManager @Inject()(s3Client: AmazonS3, clock: Clock)(implicit ec: Exe
       result.onComplete(_ => {
         // Without closing the InputStream we see large amounts of error messages:
         // Not all bytes were read from the S3ObjectInputStream
-        fileInputStream.abort()
+        fileInputStream.close()
```

Comment:
Be aware that ```close()``` can throw ```IOException```.
As there is nothing we can do in that case, we should catch and continue with the cleanup.